### PR TITLE
[TASKMGR] Hold Shift to bypass confirmation MessageBox

### PR DESCRIPTION
**Pull request by whindsaks**
Allow the user to hold shift to automatically answer `IDYES` when killing/debugging a process or changing the priority.

### CODE REVIEW

File: base/applications/taskmgr/taskmgr.c [hunk 1]
```diff
@@ -1056,6 +1056,14 @@ void TaskManager_OnTabWndSelChange(void)
     }
 }
 
+BOOL ConfirmMessageBox(HWND hWnd, LPCWSTR Text, LPCWSTR Title, UINT Type)
+{
+    UINT positive = (Type & 0xF) <= MB_OKCANCEL ? IDOK : IDYES;
```
Comment (HBelusca) [inline base/applications/taskmgr/taskmgr.c:1061]:
```suggestion
    UINT positive = ((Type & 0xF) <= MB_OKCANCEL ? IDOK : IDYES);
```

File: base/applications/taskmgr/taskmgr.c [hunk 2]
```diff
@@ -1056,6 +1056,14 @@ void TaskManager_OnTabWndSelChange(void)
...
+    if (GetKeyState(VK_SHIFT) < 0)
+        return TRUE;
+    return MessageBoxW(hWnd, Text, Title, Type) == positive;
```
Comment (HBelusca) [inline base/applications/taskmgr/taskmgr.c:1064]:
```suggestion
    return (MessageBoxW(hWnd, Text, Title, Type) == positive);
```

**Review by HBelusca — APPROVED** [review]
Useful feature in case e.g. one needs to quickly do these operations while the system is crawling like hell.

**Comment by whindsaks** [comment]
> Useful feature

NT4 has a setting in the menu to disable all confirmations but that feature was removed, probably to "protect" users.

**Pull request merged**